packages/calypso-products/src/constants/akismet.ts

```diff
@@ -0,0 +1,82 @@
+export const AKISMET_FREE = 'ak_free_yearly';
+export const AKISMET_PERSONAL_MONTHLY = 'ak_personal_monthly';
+export const AKISMET_PERSONAL_YEARLY = 'ak_personal_yearly';
```

Comment:
Shouldn't we define these constants with `PRODUCT_` prefix as well?

Reply:
Nice catch, yes they should be